.github/actions/install-deps-and-canopy/action.yml

```diff
@@ -37,8 +41,12 @@ runs:
   - name: Install dependencies
     shell: bash
     if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-    run: poetry install --no-interaction --no-root --all-extras --with dev
+    env:
```

Comment:
Do we need --with dev? By default it is with dev no?

Reply:
I wasn't sure, and I prefer to be explicit...